Update handleText()

donovian opened the pull request:
handleText() should include a few more replace calls that are needed to make text sound natural.

donovian committed:
Update handleText()

handleText() should include a few more replace calls that are needed to make text sound natural.

saworden commented:
The lines to handle "an NFT" and the like should come first. At the end there, they are useless because any instance of "NFT" has already been replaced at that point.

donovian committed:
Update content_script.js

Changed the replace order so certain outcomes will no longer never occur.

I think this also means all the plurals are unnecessary, but I didn't actually check.
